thirdeye/thirdeye-pinot/src/main/resources/assets/javascript/DataService.js

```diff
@@ -146,7 +146,7 @@ DataService.prototype = {
     fetchAnomaliesForMetricIds : function(startTime, endTime, pageNumber, metricIds, functionName, filterOnly, callback, spinner) {
       const url = constants.SEARCH_ANOMALIES_METRICIDS + startTime + this.URL_SEPARATOR + endTime + this.URL_SEPARATOR + pageNumber;
       const data = {
-        metricIds: JSON.stringify(metricIds),
+        metricIds: metricIds,
```

Comment:
Yes, this was an oversight on my part while working on encoding all endpoints (yesterday). 
I was expecting this `metricIds` to be an Array that needs to be stringified. but in actuality, the serialization had already happened.

single metric will use : `/anomalies/search/metricIds/1502780400000/1502866800000/1?metricIds=3496875&functionName=`
multiple metrics : `/anomalies/search/metricIds/1502780400000/1502866800000/1?metricIds=3496875%2C2132370&functionName=&filterOnly=true`